proxy-go/snowflake.go

```diff
@@ -171,6 +172,8 @@ func pollOffer(sid string) *webrtc.SessionDescription {
 			log.Printf("error polling broker: %s", err)
 		} else {
 			defer resp.Body.Close()
+			nextPoll, _ := strconv.Atoi(resp.Header.Get("Snowflake-Next-Poll"))
+			pollInterval = time.Duration(nextPoll) * time.Second
```

Comment:
It looks like this won't be used until *after* the next poll.  Maybe you need to move the `timeOfNextPoll` calculation to the end of the loop?

Reply:
You are right, I completely skipped that. Will fix this and other things in the next commit.